server/serial_console.py

```
@@ -0,0 +1,175 @@
+            port_id = data.get("portId", "")
+            baudrate = int(data.get("baudrate", 115200))
+            if not port_id or baudrate <= 0:
+                raise ValueError("A serial port and positive baud rate are required")
+
+            await asyncio.to_thread(console.open, port_id, baudrate)
+            if not console.reader_started:
+                console.reader_started = True
+                sio.start_background_task(serial_console_read_loop, console, sio)
+            await sio.emit("serialConsoleStatus", console.info())
+            return {"status": "OK"}
+        except Exception as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)